ni_measurement_plugin_sdk_service/_internal/parameter/metadata.py

```diff
@@ -54,18 +54,12 @@ def validate_default_value_type(parameter_metadata: ParameterMetadata) -> None:
     if default_value is None:
         return None
 
-    expected_type = type(
-        serialization_strategy.get_type_default(
-            parameter_metadata.type, parameter_metadata.repeated
-        )
-    )
+    expected_type = type(get_type_default(parameter_metadata.type, parameter_metadata.repeated))
     display_name = parameter_metadata.display_name
     enum_values_annotation = get_enum_values_annotation(parameter_metadata)
 
     if parameter_metadata.repeated:
-        expected_element_type = type(
-            serialization_strategy.get_type_default(parameter_metadata.type, False)
-        )
+        expected_element_type = type(get_type_default(parameter_metadata.type, False))
```

Comment:
I'm not sure why it's there since it was already there before I changed anything, although I think it's to get the class from get_type_default() since it would return 0, 0.0, "", or  False.  So I didn't intend it to make a copy of the function or anything like that.

Reply:
Nevermind, I was misreading this. `type(x)` queries the type of `x`. I was reading it as if `type` was a variable that contained a type.